### PR TITLE
Added the Gorilla API Zoo contribution

### PR DESCRIPTION
**Pull request by tianjunz**
Added the Gorilla API Zoo contribution

### CODE REVIEW

File: data/README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+Wants Gorilla to use your API? Wants to train your own Gorilla with the dataset? Learn about the entire workflow and how to contribute to Gorilla API App Store! Gorilla API App Store intends to enhance LLM's capability to use tools through API calls. We appreciate everyone's effort and contribution!
+
+## How to Contribute?
+
+Contribute to Gorilla API Store is very easy! 
+
+1. **JSON Contribution**: It only takes two steps
+
+- Step 1: Submit an API JSON file or a URL JSON file following our cirteria.
```
Comment (firtoz) [inline data/README.md:13]:
```suggestion
- Step 1: Submit an API JSON file or a URL JSON file following our criteria. 
```

File: data/README.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+### URL JSON
+
+We also provide you another much simpler approach for you to contribute! Provide a simple url to your API documentation, we'll process for you. For this, submit a JSON file containing the URL: 
+
+| Field      |  Type  | Description/Options     | Required |
+| :---       | :----: |          :----         |   :---:   |
+| user_name     | String       | Name of User   | ✅ |
+| api_name      | String       | Name of API (maximum 20 words)   | ✅ |
+| api_url      | String       | URL to API documentation   | ✅ |
+| Questions | List[String] | A question describing a real-life scenario that uses this API. Please donnot include specific API name. | [Optional]:fire: |
```
Comment (firtoz) [inline data/README.md:94]:
```suggestion
| Questions | List[String] | A question describing a real-life scenario that uses this API. Please do not include specific API name. | [Optional]:fire: |
```

**Comment by ShishirPatil** [comment]
@firtoz Thanks for proactively reviewing our PR! Really excited to have you contribute to the project.

File: data/README.md (hunk 1 above)
Comment (ShishirPatil) [reply data/README.md]: File evolved

**Review by ShishirPatil — APPROVED** [review]

**Pull request merged**